problems/countThrees.js

```diff
@@ -12,9 +12,17 @@
 * ex: countThrees(47)
 * returns: 0
 */
+function countThrees(n, num) {
```

Comment:
reread this question, it's asking how many 3's appear in the number. We can achieve this by looping through each digit and if the digit is a 3 then add one to a counter.